features/page_objects/journey/email_page.rb

```diff
@@ -1,11 +1,11 @@
 # frozen_string_literal: true
 
-class ApplicantEmailPage < SitePrism::Page
+class EmailPage < SitePrism::Page
 
   element(:error, ".error-summary")
 
-  element(:email, "#applicant_email_form_applicant_email")
-  element(:confirm_email, "#applicant_email_form_confirmed_email")
+  element(:email, "#applicant_email_form_applicant_email,#contact_email_form_contact_email")
```

Comment:
I assume this is ok as the tests are passing but it looks wrong. Seems this `#contact_email_form_contact_email` would fail if run against the applicant email page.

Reply:
On reflection I'm guessing this is some SitePrism wizardry you're going to tell me about 😁

Reply:
This is something Cinti helped me out with - this was trial and error. The comma seems to act as an OR operator, and this looked like the only way I could get a page object which worked for both the applicant and contact email. I'm struggling to find a link explaining this technique, but the tests pass :)

Reply:
Yeah, I think its actually a CSS selector trick than anything with SitePrism. So don't worry about it, TIL!